File: src/test/OLD_TEST_MATLAB/checkOutput.m
function[]=checkOutput(varName, matrix, refMatrix, tol)

	fprintf('Checking %s\n',varName);
	maxError = max(max(max(max(abs(matrix - refMatrix)))));
	if maxError >= tol
		fprintf('    Inaccuracy: max(abs(error)) = %g >= %g\n',maxError,tol);
	else
		fprintf('    OK\n');
	end

end

File: src/test/generate_test_data/R_test_helpers/checkOutput.R
checkOutput = function(varName, matrix, refMatrix, tol){
  print('Checking %s\n',varName);
  maxError = max(max(max(max(abs(matrix - refMatrix)))));
  if(maxError >= tol){
    print('Inaccuracy: max(abs(error)) = %g >= %g\n',maxError,tol)
  }
  else{
    print('OK\n')
  }
}